presto-main/src/main/java/com/facebook/presto/sql/ExpressionUtils.java

```diff
@@ -173,6 +185,18 @@ public static Expression stripDeterministicConjuncts(Expression expression)
                 .collect(toImmutableList()));
     }
 
+    public static LogicalBinaryExpression.Type flipLogicalBinaryExpressionType(LogicalBinaryExpression.Type type)
```

Comment:
It is similar to `flipComparison` method, so it's better to be consistent I think